index.d.ts

```diff
@@ -68,6 +68,12 @@ declare const electronDebug: {
 	@param window - Default: `BrowserWindow.getFocusedWindow()`
 	*/
 	openDevTools(window?: BrowserWindow): void;
+
+	/**
+	The absolute path to a preload file that will enable `devtron`.
+	Use it in a call to [`session#setPreloads`](https://www.electronjs.org/docs/api/session#sessetpreloadspreloads).
```

Comment:
This should be in sync with the readme.

Reply:
Thanks for the phrasing help. I updated the documentation.